### ISSUE TITLE
ActiveSpan is too Opaque

### ISSUE THREAD

**Issue by rbtcollins** [opentracing/opentracing-java#208]
We'd like to be able to use the TraceId from the  ActiveSpan in three additional contexts

[] In external logs, so we can silo sensitive data from the widely-accessed tracing console but still correlate
[] In response headers, so that we can do browser integration and directly link to the resulting trace from our UI for developers
[] In request headers when using external APIs that support correlation e.g with x-request-id, so that we can ask their support to examine a particular request.

Right now we cannot because traceId is considered an implementation detail, not part of the contract.

**Comment by devinsba** [comment]
There is an ongoing discussion of this: https://github.com/opentracing/specification/issues/24

**Comment by tedsuo** [comment]
I agree @rbtcollins, I want to push on this once https://github.com/opentracing/opentracing-java/issues/189 is over.

**Comment by cwe1ss** [comment]
fyi, this is currently getting specified with https://github.com/opentracing/specification/pull/109

**Comment by carlosalberto** [comment]
Wondering if we should close this Issue, as there's already an ongoing discussion (as Christian pointed out), as well as #280?

@rbtcollins ok to close it? ;)
